docs/QueryExamples.rst

```diff
@@ -31,3 +31,101 @@ And it is straightforward, but it won't work.  Because Cypher matches one subgra
       .where_not('ID(other_contact) IN first_level_ids')
 
 And there we have a query which is much more verbose than the original code, but accomplishes the goal in a single query.  Having two queries isn't neccessarily bad, so the code's complexity should be weighed against how both versions perform on real datasets.
+
+
+Example 2: Simple Recommendation Engine
+---------------------------------------
+
+If you are interested in more complex collaborative filter methods check out this article https://neo4j.com/blog/collaborative-filtering-creating-teams/
```

Comment:
I'd change this reStructuredText to be
```rst
`article <https://neo4j.com/blog/collaborative-filtering-creating-teams/>`_
```
so that you just get an anchor tag on the intended word.